ui/src/components/logs/TaskRunDetails.vue

```diff
@@ -44,6 +44,7 @@
                         :min-item-size="1"
                         key-field="index"
                         class="log-lines"
+                        :style="{maxHeight: currentTaskRuns.length === 1 ? 'calc(100vh - 350px)' : '50vh'}"
```

Comment:
Could this be made an extra class?
It has the advantage of allowing a design to play with it in the browser dev tools.

```suggestion
                        :class="{'single-line': currentTaskRuns.length === 1}"
```